lib/src/style/QlementineStyle.cpp

```diff
@@ -3983,7 +3983,7 @@ int QlementineStyle::styleHint(StyleHint sh, const QStyleOption* opt, const QWid
     case SH_ProgressDialog_CenterCancelButton:
       return false;
     case SH_ProgressDialog_TextLabelAlignment:
-      return Qt::AlignLeft;
+      return Qt::AlignCenter;
```

Comment:
I would suggest to rather do the change in our style subclass, in `WinAdminStyle::styleHint()`, because this is more customization on our side that an upstream bug.

Reply:
yes, agree